spec/setup_acceptance_node.pp

```diff
@@ -0,0 +1,37 @@
+$major = $facts['os']['release']['major']
+
+case $major {
+  '7': {
+    package { 'centos-release-scl-rh':
+      ensure => installed,
+    }
+
+    package { 'rh-redis5-redis':
+      ensure  => installed,
+      require => Package['centos-release-scl-rh'],
+    }
+  }
+  '8': {
+    package { 'glibc-langpack-en':
```

Comment:
I like the approach of splitting off all module specific acceptance config into a new file.

I want to say, could we add a comment that installing `glibc-langpack-en` will no longer be necessary once we have: https://github.com/puppetlabs/beaker-hostgenerator/pull/183

But I could tackle that in https://github.com/theforeman/puppet-pulpcore/pull/77 after rebasing, since I assume I will need to make some further changes in this file to get EL8 working regardless.

Reply:
the language pack is only needed if we need PostgreSQL, so I opted to keep it in a local override.